andela/ah-frontend-odin
# #161340144 Add articles comments

#### What does this PR do?
This PR enables users to see and add  comments to articles
#### Description of Task to be completed?
- List all article comments with their authors
- Allow only authenticated users to add comments to articles
#### How should this be manually tested?
- Refer to the `README.md` for instructions on how to get up and running with this application.
- With the application served and running in the browser, ensure you have atleast one article created
- Click on the `Readmore...` to access the article's detail page
- Login in using one of the available authentication methods and you will see a form for adding a comment
- Add a comment and hit the comment button, the new comment should instantly be listed in the article's comment.
#### What are the relevant pivotal tracker stories?
[#161340144](https://www.pivotaltracker.com/story/show/161340144)
#### Screenshots
![comment](https://user-images.githubusercontent.com/33207024/50197924-681a3a00-035a-11e9-834f-397756e86d17.png)


#### Questions:
